tests/utils/constants/config.go

```diff
@@ -92,3 +94,16 @@ func GetEnvWithDefault(key string, defaultValue string) string {
 	}
 	return defaultValue
 }
+
+func GetRhcsOutputDir() string {
+	var rhcsNewOutPath string
+
+	if GetEnvWithDefault("RHCS_OUTPUT", "") != "" {
+		rhcsNewOutPath = os.Getenv("RHCS_OUTPUT")
+		os.MkdirAll(rhcsNewOutPath, 0777)
```

Comment:
If we have this env RHCS_OUTPUT set means there is a dir, do not need to create again.